### ISSUE TITLE
Nightly version icon is not adaptive

### ISSUE THREAD

**Issue by flexagoon** [cgeo/cgeo#14795]
### Describe your problem!

The icon of the Nightly build of cgeo doesn't adapt to the wallpaper colors, even though the icon of the release version does.

### How to reproduce?

1. Enable themed icons on your phone
2. Install the nightly build of cgeo 

### Actual result after these steps?

The icon is the default c:geo icon

### Expected result after these steps?

Cgeo uses the themed icon

### Reproducible

Yes

### c:geo Version

2023.11.04-NB-146a8bc

**Comment by ztNFny** [comment]
non-release-icons are not intended to be adaptive - so works as designed

**Comment by flexagoon** [comment]
> non-release-icons are not intended to be adaptive

@ztNFny why is that? I don't really get why a basic Android feature should be supported on the release version but not on the nightly version

**Comment by ztNFny** [comment]
Only release icon is adaptive by design, making the other 3 adaptive is a feature request, not a bug.
If anyone wants to create those 3 icons go ahead ...
